dluhc-component-library/src/components/maps/MapComponent.tsx

```diff
@@ -18,6 +22,9 @@ interface MapComponentProps {
   style?: CSSProperties;
   value?: Boundary;
   onChange?: (boundary: Boundary) => void;
+  submittedSites?: Boundary[];
```

Comment:
Is there a reason this cant be `ReadonlyArray<Boundary>`?

These should also not be called `submittedSites`. That is use case specific where we are writing a generic component. Any list of boundaries can be passed here. We should rename it to something like `boundaries` or `areas` etc